modules/v2x/proto/v2x_service_car_to_obu.proto

```diff
@@ -1,17 +1,12 @@
-
 syntax = "proto2";
 
 package apollo.v2x;
 
-import "modules/perception/proto/perception_obstacle.proto";
-import "modules/v2x/proto/v2x_carstatus.proto";
+import "modules/v2x/proto/v2x_car_status.proto";
 
 service CarToObu {
-  rpc PushCarStatus(apollo.v2x.CarStatus) returns (UpdateStatus) {
-  }
-  rpc PushPerceptionResult(apollo.perception.PerceptionObstacles)
-      returns (UpdateStatus) {
-  }
+  rpc PushCarStatus(apollo.v2x.CarStatus) returns (UpdateStatus)
+  {}
```

Comment:
I would suggest put the openning bracket to Line 8. Our `scripts/proto_build_generator.py` maybe not smart enough.

Reply:
this change is fit for ```proto_build_generator.py``` include space, \n and so on.